packages/ed25519-signature-2018/scripts/issuanceDate-generate.js

```diff
@@ -0,0 +1,186 @@
+const {
+  Ed25519VerificationKey2018,
+} = require("@digitalbazaar/ed25519-verification-key-2018");
+const {
+  Ed25519Signature2018,
+} = require("@digitalbazaar/ed25519-signature-2018");
+
+const fs = require("fs");
+const path = require("path");
+
+const jsigs = require("jsonld-signatures");
+const { AssertionProofPurpose } = jsigs.purposes;
+const moment = require("moment");
+const documentLoader = require("./documentLoader");
+const credential = require("../src/__fixtures__/credentials/case-7.json");
+const rawKeyJson = require("../src/__fixtures__/keys/key.json");
+
+const ISSUED_ON = new Date("1996-09-29T15:01:23.456Z").getTime();
+const CREATED_ON = new Date("2021-10-15T15:01:23.456Z").getTime();
+
+/* Abstract:
+For the `issuanceDate` test, we first want to create a set of 
+signed credentials with proofs with different `issuanceDates`
+for a fixed time. We then want to regenerate these proofs
+to make sure that we get the same jws, and ideally the exact
+same proofs with the same `created` date
+*/
+
+const TESTS = [
+  // Unset conditions
+  "removed",
+  undefined,
+  null,
+  0,
+  "",
+  // Int conditions
+  1635774995208,
+  -1635774995208,
+  1,
+  -1,
+  123,
+  -123,
+  12345,
+  -12345,
+  // String conditions
+  moment(ISSUED_ON).format(),
+  moment(ISSUED_ON).format("dddd, MMMM Do YYYY, h:mm:ss a"),
+  moment(ISSUED_ON).format("Do dddd MMMM gggg"),
+  moment(ISSUED_ON).format("MMMM Qo DD YYYY"),
+  moment(ISSUED_ON).format("D MMM YYYY"),
+  moment(ISSUED_ON).format("YYYY-MM-DD"),
+  moment(ISSUED_ON).format("ddd, DD MMM YYYY HH:mm:ss z"),
+  moment(ISSUED_ON).format("MM DD YYYY"),
+  moment(ISSUED_ON).format("MMM D, YYYY"),
+  "1996-09-29T15:01:23",
+  "1996-09-29T15:01:23.456+09:00",
+  "1996-09-29T15:01:23.456Z",
+  "1996-09-29T15:01:23Z",
+  "1996-09-29T15:01:23-05:00",
+  moment(ISSUED_ON).toArray(),
+  // moment(ISSUED_ON).toObject(),
+];
+
+/*
+Note: To account for the posibility of different timezones,
```

Comment:
agree, but this sounds like an issue, you should raise an issue and link it here, if you think it "should be done in the future"... avoid leaving comments in code that "request work" that is not described in an issue.

Reply:
To avoid getting ahead of myself, i took this out of the code to test as-is. If it could possibly be an issue later, then I will create an issue for it.